tests/phpunit/class-wc-stripe-payment-gateway-test.php

```
@@ -1031,4 +1031,91 @@ public function provide_test_disable_subscription_edit_for_india(): array {
+			],
+			'add mode - first time fee setting'             => [
+				'existing_fee' => 0,
+				'existing_net' => 0,
+				'api_fee'      => 100,
+				'api_net'      => 4900,
+				'replace'      => false,
+				'expected_fee' => 1.00,
+				'expected_net' => 49.00,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_update_fees_scenarios is not named in camelCase. (undefined)

(CamelCaseMethodName)